src/Tools/dotnet-dsrouter/ADBTcpRouterFactory.cs

```diff
@@ -166,7 +187,7 @@ public override void Start()
                 _ownsPortReverse = false;
                 Logger.LogError("Failed setting up adb port reverse." +
                     " This might lead to problems communicating with Android application." +
-                    " Make sure env variable ANDROID_SDK_ROOT is set and points to an Android SDK." +
+                    " Make sure env variable $ANDROID_HOME is set and points to an Android SDK." +
```

Comment:
we should probably update our docs as well, https://learn.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-dsrouter, it reference ANDROID_SDK_ROOT  as well.

Reply:
This one updates the docs to match this PR:
* https://github.com/dotnet/docs/pull/47567